src/games/strategy/triplea/TripleA.java

```diff
@@ -90,7 +92,7 @@ public void shutDown() {
 
 
   @Override
-  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless) {
+  public void startGame(final IGame game, final Set<IGamePlayer> players, final boolean headless, Runnable exceptionAction) {
```

Comment:
Instead of a 'Runnable exceptionAction', what if we instead returned 'true/false' to indicate if the game started? Would that simplify things?

It also seems the headless instance will need cleanup, a default action of no-op for cleanup does not seem right for all cases.